apps/server/src/modules/files/repo/orphaned-files.repo.ts

```diff
@@ -54,6 +54,50 @@ const tasksQuery = [
 export class OrphanedFilesRepo {
 	constructor(protected readonly _em: EntityManager) {}
 
+	async findDuplicatedFileRecords(parentType: FileRecordParentType) {
+		const fileRecords: FileRecord[] = [];
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		let results: any[] = [];
+		if (parentType === FileRecordParentType.Lesson) {
```

Comment:
This work for lessons, but needwe also implementations for task we should add a if(..Task) { throw NotImplemented() }.